scripts/check_phase44c_projection_root_probe.py

```
@@ -0,0 +1,279 @@
+import re
+import sys
+from typing import Any, Iterable
+
+
+SCHEMA = "phase44c-projection-root-binding-manifest-v1"
+PROBE = "phase44c-source-emitted-projection-root-binding"
+PHASE43_SURFACE_VERSION = "phase43-history-replay-field-projection-v1"
+HASH32_RE = re.compile(r"^[0-9a-f]{64}$")
+DEFAULT_MANIFEST = pathlib.Path("docs/engineering/design/phase44c-projection-root-manifest.json")
+DEFAULT_EVIDENCE = pathlib.Path("target/phase44c-projection-root-probe/evidence.json")
+DEFAULT_STWO_ROOT = pathlib.Path("/tmp/zkai-research/repos/stwo")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/zkai-research/repos/stwo"

(S108)